### PR TITLE
Remove ISchema in MultiClassClassifierScorer.cs

### PR DESCRIPTION
**Pull request by wschin**
This PR completely remove `ISchema` and its implementation used in `MultiClassClassifierScore`. It's a part of #1501.

### CODE REVIEW

File: src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs [hunk 1]
```diff
@@ -292,9 +291,31 @@ public ISchemaBindableMapper Bindable
...
+                {
+                    var builder = new SchemaBuilder();
+                    for (int i = 0; i < partialSchema.ColumnCount; ++i)
+                    {
+                        var meta = new MetadataBuilder();
+                        // Take all existing metadata
+                        meta.Add(partialSchema[i].Metadata, selector: s => true);
+                        // Add label names
+                        if (i == scoreColumnIndex)
+                            meta.Add(labelNameKind, labelNameType, labelNameGetter);
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs]:
```   if (col == _scoreCol && kind == _metadataKind)
                        {
                            _labelNameGetter.Marshal(0, ref value);
                            return;
                        }
```
this is from original code.

make same if statement and if it's not true then just copy metadata from source.
if it's true, modify your selector for everything is true except metadatakind, and add you labelNameGetter to metadata.

Comment (wschin) [reply src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs]: Ok. Just pushed a fix. Please let me know if I did that correctly.

**Comment by sfilipi** [comment]
>                     return _mapper.GetDependencies(predicate);

arrow style for single line returns:
=> 

---
Refers to: src/Microsoft.ML.Data/Scorers/MultiClassClassifierScorer.cs:331 in e5ecf1c. [](commit_id = e5ecf1c74e10438e804b5a9b7beaba1853923068, deletion_comment = False)

**Comment by wschin** [comment]
@sfilipi, Thanks. I fixed similar problems in that file.

**Pull request merged**